xbmc/windows/GUIWindowScreensaverDim.cpp

```diff
@@ -40,11 +40,9 @@ CGUIWindowScreensaverDim::~CGUIWindowScreensaverDim(void)
 void CGUIWindowScreensaverDim::UpdateVisibility()
 {
   float level = g_application.GetDimScreenSaverLevel();
+  m_dimLevel = level;
```

Comment:
I'd make it `m_dimLevel = g_application.GetDimScreenSaverLevel(); if (m_dimlevel) ..`.
Really just a minor and more a personal taste. Thanks for checking and fixing that ugly issue!